src/main.js

```diff
@@ -1,19 +1,137 @@
 
-// Add your event listeners here 👇
 
 
-// Create your event handlers and other functions here 👇
+function changeFormView() {
+  homeView.classList.add('hidden');
+  formView.classList.remove('hidden');
+  viewSavedView.classList.add('hidden');
+  saveCoverButton.classList.add('hidden');
+  randomCoverButton.classList.add('hidden');
+  homeButton.classList.remove('hidden');
+}
+
+function changeViewSavedView() {
+  randomCoverButton.classList.add('hidden');
+  saveCoverButton.classList.add('hidden');
+  homeButton.classList.remove('hidden');
+  viewSavedView.classList.remove('hidden');
+  homeView.classList.add('hidden');
+  formView.classList.add('hidden');
+}
+
+function changeHomeView() {
+  homeButton.classList.add('hidden');
+  randomCoverButton.classList.remove('hidden');
+  saveCoverButton.classList.remove('hidden');
+  homeView.classList.remove('hidden');
+  formView.classList.add('hidden');
+  viewSavedView.classList.add('hidden');
+}
+
+function makeRandomCover() {
+  currentCover = new Cover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)]);
+  coverImage.src = currentCover.cover;
+  title.innerText = currentCover.title;
+  descriptor1.innerText = currentCover.tagline1;
+  descriptor2.innerText = currentCover.tagline2;
+}
+
+function createNewBook() {
+  covers.push(inputCover.value);
+  titles.push(inputTitle.value);
+  descriptors.push(inputFirstDescriptor.value);
+  descriptors.push(inputSecondDescriptor.value);
+  console.log(inputSecondDescriptor.value);
+}
 
+function showNewBookHome() {
+  currentCover = new Cover(inputCover.value, inputTitle.value, inputFirstDescriptor.value, inputSecondDescriptor.value);
+  coverImage.src = currentCover.cover;
+  title.innerText = currentCover.title;
+  descriptor1.innerText = currentCover.tagline1;
+  descriptor2.innerText = currentCover.tagline2;
+  changeHomeView();
+}
+
+function saveCurrentCover() {
+  if (!savedCovers.includes(currentCover)) {
+    savedCovers.push(currentCover);
+    showSavedCovers();
+  }
+}
+
+function showSavedCovers() {
+    savedCoversSection.innerHTML += `
+      <div class="mini-cover">
+        <img class="mini-cover" src="${currentCover.cover}">
+        <h1 class="cover-title">${currentCover.title}</h1>
+        <h2 class="tagline">A tale of ${currentCover.tagline1} and ${currentCover.tagline2}</h2>
+      </div>
+    `
+  }
+
+function deleteSavedCover() {
+  if (event.target.className === 'mini-cover') {
+    var div = event.target.parentElement;
+    savedCoversSection.removeChild(div);
+  }
+}
+
+function saveCustomBook() {
+  event.preventDefault();
+  if (inputCover.value && inputTitle.value && inputFirstDescriptor.value && inputSecondDescriptor.value) {
+    createNewBook();
+    showNewBookHome();
+  }
+}
 
-// We've provided one function to get you started
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
```

Comment:
This a great, clean way to generate a random index within your array's range. Nice work! 😎